### ISSUE TITLE
Support ##< for documenting members and variables

### ISSUE THREAD

**Issue by marknelsonengineer** [doxygen/doxygen#9911]
Putting documentation after members is very helpful.  The feature is documented here: https://www.doxygen.nl/manual/docblocks.html#memberdoc

I'd like to extend this support to Python code.  Specifically, I'd like the following code to work:

````
OUTPUT_FILE = "STATISTICS.md"    ##< All output is redirected to this file
````

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #9928

Example: [example.tar.gz](https://github.com/doxygen/doxygen/files/11010807/example.tar.gz)

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.7.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by marknelsonengineer** [comment]
@albert-github and @doxygen.  I wanted to say **thank you** both for this fix.  I have verified that it works on my code base.

Overall, the update looks good.
